src/backend/distributed/metadata/node_metadata.c

```diff
@@ -757,6 +758,20 @@ master_update_node(PG_FUNCTION_ARGS)
 		if (force)
 		{
 			handle = StartLockAcquireHelperBackgroundWorker(MyProcPid, lock_cooldown);
+			if (!handle)
+			{
```

Comment:
I wonder if we should have a debug/notice message here saying that we will fall back to normal `master_update_node` even though `force` option was given so that this can give us some info in production hopefully.

Reply:
I think it might worth to give a warning similar to that we're removing in this pr

```c
ereport(WARNING, (errmsg("could not start lock acquiring background worker to "
                         "force the update. Degrading to acquiring locks "
                         "with a lock time out."),
                  errhint("Increasing max_worker_processes might help.")));
```